app/views/layouts/portal.html.erb

```diff
@@ -39,6 +39,10 @@ By default, it renders:
     <% else %>
       <title><%= @portal.page_title%></title>
     <% end %>
+    
+    <% if @portal.logo.present? %>
+      <link rel="icon" type="image/png" href="<%= url_for(@portal.logo) %>">
```

Comment:
@sojan-official is there a way to know the mimetype of the image?

Reply:
Type is not required.